src/react-intl-messages.ts

```diff
@@ -653,6 +657,7 @@ export default {
         [FM.ACTIONCREATOREDITOR_TRAINDIALOGSBUTTON_TEXT]: 'Train Dialogs',
         [FM.ACTIONCREATOREDITOR_CHECKBOX_TERMINAL_LABEL]: 'Wait for Response?',
         [FM.ACTIONCREATOREDITOR_CHECKBOX_REPROMPT_LABEL]: 'Reprompt on unexected user input',
+        [FM.ACTIONCREATOREDITOR_CHECKBOX_ENTRY_NODE_LABEL]: 'Direct unknown user utterances to this action',
```

Comment:
Still think it's weird to mix "input" and "utterance" in same product. Because they're named different people could assume they refer to different things but they are the interchangeable. I prefer "input" as it seems like a more layman term and doesn't imply audio/voice like utterance does but I supposed we'rd already inconsistent on this won't matter.

Reply:
Updating in new PR.  Note that there are other existing strings that reference "utterance", not modifying those.